<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>[PI] Runetoplist script v2 [PI]</title>
    <link>https://forum.moparisthebest.com/t/pi-runetoplist-script-v2-pi/453807</link>
    <description>i have been working days now for finding out how this works, and finally i got it to work! and i was happy untill that a player used the command ::claim, wich made all the players of the server lagg out, and after 30 seconds they could log back in. does anyone know how to fix it?

[code]The last packet successfully received from the server was 1.344.662.672.572 mill
iseconds ago.  The last packet sent successfully to the server was 0 millisecond
s ago.
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance
0(Native Method)
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance
(NativeConstructorAccessorImpl.java:39)
[11-08-12 07:24]:       at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(DelegatingConstructorAccessorImpl.java:27)
[11-08-12 07:24]:       at java.lang.reflect.Constructor.newInstance(Constructor
.java:513)
[11-08-12 07:24]:       at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[11-08-12 07:24]:       at com.mysql.jdbc.SQLError.createCommunicationsException
(SQLError.java:1074)
[11-08-12 07:24]:       at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI
mpl.java:2214)
[11-08-12 07:24]:       at com.mysql.jdbc.ConnectionImpl.&lt;init&gt;(ConnectionImpl.j
ava:773)
[11-08-12 07:24]:       at com.mysql.jdbc.JDBC4Connection.&lt;init&gt;(JDBC4Connection
.java:46)
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance
0(Native Method)
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance
(NativeConstructorAccessorImpl.java:39)
[11-08-12 07:24]:       at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(DelegatingConstructorAccessorImpl.java:27)
[11-08-12 07:24]:       at java.lang.reflect.Constructor.newInstance(Constructor
.java:513)
[11-08-12 07:24]:       at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[11-08-12 07:24]:       at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionI
mpl.java:352)
[11-08-12 07:24]:       at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegist
eringDriver.java:282)
[11-08-12 07:24]:       at java.sql.DriverManager.getConnection(DriverManager.ja
va:582)
[11-08-12 07:24]:       at java.sql.DriverManager.getConnection(DriverManager.ja
va:185)
[11-08-12 07:24]:       at org.runetoplist.MySQLConnectionPool.getFreeConnection
(MySQLConnectionPool.java:61)
[11-08-12 07:24]:       at org.runetoplist.MySQLResultCallable.call(MySQLResultC
allable.java:29)
[11-08-12 07:24]:       at org.runetoplist.MySQLResultCallable.call(MySQLResultC
allable.java:1)
[11-08-12 07:24]:       at java.util.concurrent.FutureTask$Sync.innerRun(FutureT
ask.java:303)
[11-08-12 07:24]:       at java.util.concurrent.FutureTask.run(FutureTask.java:1
38)
[11-08-12 07:24]:       at java.util.concurrent.ThreadPoolExecutor$Worker.runTas
k(ThreadPoolExecutor.java:886)
[11-08-12 07:24]:       at java.util.concurrent.ThreadPoolExecutor$Worker.run(Th
readPoolExecutor.java:908)
[11-08-12 07:24]:       at java.lang.Thread.run(Thread.java:662)
[11-08-12 07:24]: Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsExcep
tion: Communications link failure

The last packet successfully received from the server was 1.344.662.672.572 mill
iseconds ago.  The last packet sent successfully to the server was 0 millisecond
s ago.
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance
0(Native Method)
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance
(NativeConstructorAccessorImpl.java:39)
[11-08-12 07:24]:       at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(DelegatingConstructorAccessorImpl.java:27)
[11-08-12 07:24]:       at java.lang.reflect.Constructor.newInstance(Constructor
.java:513)
[11-08-12 07:24]:       at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[11-08-12 07:24]:       at com.mysql.jdbc.SQLError.createCommunicationsException
(SQLError.java:1074)
[11-08-12 07:24]:       at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:341)
[11-08-12 07:24]:       at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI
mpl.java:2137)
[11-08-12 07:24]:       ... 19 more
[11-08-12 07:24]: Caused by: java.net.ConnectException: Connection timed out: co
nnect
[11-08-12 07:24]:       at java.net.PlainSocketImpl.socketConnect(Native Method)

[11-08-12 07:24]:       at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.ja
va:351)
[11-08-12 07:24]:       at java.net.PlainSocketImpl.connectToAddress(PlainSocket
Impl.java:213)
[11-08-12 07:24]:       at java.net.PlainSocketImpl.connect(PlainSocketImpl.java
:200)
[11-08-12 07:24]:       at java.net.SocksSocketImpl.connect(SocksSocketImpl.java
:366)
[11-08-12 07:24]:       at java.net.Socket.connect(Socket.java:529)
[11-08-12 07:24]:       at java.net.Socket.connect(Socket.java:478)
[11-08-12 07:24]:       at java.net.Socket.&lt;init&gt;(Socket.java:375)
[11-08-12 07:24]:       at java.net.Socket.&lt;init&gt;(Socket.java:218)
[11-08-12 07:24]:       at com.mysql.jdbc.StandardSocketFactory.connect(Standard
SocketFactory.java:253)
[11-08-12 07:24]:       at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:290)
[11-08-12 07:24]:       ... 20 more
[11-08-12 07:24]: java.util.concurrent.ExecutionException: java.lang.NullPointer
Exception
[11-08-12 07:24]:       at java.util.concurrent.FutureTask$Sync.innerGet(FutureT
ask.java:222)
[11-08-12 07:24]:       at java.util.concurrent.FutureTask.get(FutureTask.java:8
3)
[11-08-12 07:24]:       at org.runetoplist.VoteChecker.getReward(VoteChecker.jav
a:40)
[11-08-12 07:24]:       at server.model.players.packets.Commands.playerCommands(
Commands.java:658)
[11-08-12 07:24]:       at server.model.players.packets.Commands.processPacket(C
ommands.java:52)
[11-08-12 07:24]:       at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:137)
[11-08-12 07:24]:       at server.model.players.Client.processQueuedPackets(Clie
nt.java:3539)
[11-08-12 07:24]:       at server.model.players.PlayerHandler.process(PlayerHand
ler.java:274)
[11-08-12 07:24]:       at server.GameEngine.run(GameEngine.java:27)
[11-08-12 07:24]:       at server.Server$1.execute(Server.java:183)
[11-08-12 07:24]:       at server.task.Task.tick(Task.java:107)
[11-08-12 07:24]:       at server.task.TaskScheduler.run(TaskScheduler.java:125)

[11-08-12 07:24]:       at java.util.concurrent.Executors$RunnableAdapter.call(E
xecutors.java:441)
[11-08-12 07:24]:       at java.util.concurrent.FutureTask$Sync.innerRunAndReset
(FutureTask.java:317)
[11-08-12 07:24]:       at java.util.concurrent.FutureTask.runAndReset(FutureTas
k.java:150)
[11-08-12 07:24]:       at java.util.concurrent.ScheduledThreadPoolExecutor$Sche
duledFutureTask.access$101(ScheduledThreadPoolExecutor.java:98)
[11-08-12 07:24]:       at java.util.concurrent.ScheduledThreadPoolExecutor$Sche
duledFutureTask.runPeriodic(ScheduledThreadPoolExecutor.java:180)
[11-08-12 07:24]:       at java.util.concurrent.ScheduledThreadPoolExecutor$Sche
duledFutureTask.run(ScheduledThreadPoolExecutor.java:204)
[11-08-12 07:24]:       at java.util.concurrent.ThreadPoolExecutor$Worker.runTas
k(ThreadPoolExecutor.java:886)
[11-08-12 07:24]:       at java.util.concurrent.ThreadPoolExecutor$Worker.run(Th
readPoolExecutor.java:908)
[11-08-12 07:24]:       at java.lang.Thread.run(Thread.java:662)
[11-08-12 07:24]: Caused by: java.lang.NullPointerException
[11-08-12 07:24]:       at org.runetoplist.PoolConnection.borrow(PoolConnection.
java:27)
[11-08-12 07:24]:       at org.runetoplist.MySQLConnectionPool.getFreeConnection
(MySQLConnectionPool.java:66)
[11-08-12 07:24]:       at org.runetoplist.MySQLResultCallable.call(MySQLResultC
allable.java:29)
[11-08-12 07:24]:       at org.runetoplist.MySQLResultCallable.call(MySQLResultC
allable.java:1)
[11-08-12 07:24]:       at java.util.concurrent.FutureTask$Sync.innerRun(FutureT
ask.java:303)
[11-08-12 07:24]:       at java.util.concurrent.FutureTask.run(FutureTask.java:1
38)
[11-08-12 07:24]:       ... 3 more[/code]

i&#39;ll pay 40M rsps gold, i go first but i must see that you have vouches.</description>
    
    <lastBuildDate>Sun, 12 Aug 2012 01:15:51 +0000</lastBuildDate>
    <category>Runescape</category>
    <atom:link href="https://forum.moparisthebest.com/t/pi-runetoplist-script-v2-pi/453807.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>[PI] Runetoplist script v2 [PI]</title>
        <dc:creator><![CDATA[@24jared24 24jared24]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/24jared24">@24jared24</a> wrote:</p>
          <blockquote>
              <p>I have the same problem, so bump <img src="https://forum.moparisthebest.com/images/emoji/twitter/slight_smile.png?v=5" title=":slight_smile:" class="emoji" alt=":slight_smile:"></p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-runetoplist-script-v2-pi/453807/2">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-runetoplist-script-v2-pi/453807/2</link>
        <pubDate>Sun, 12 Aug 2012 01:15:51 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-453807-2</guid>
        <source url="https://forum.moparisthebest.com/t/pi-runetoplist-script-v2-pi/453807.rss">[PI] Runetoplist script v2 [PI]</source>
      </item>
      <item>
        <title>[PI] Runetoplist script v2 [PI]</title>
        <dc:creator><![CDATA[@ownedin3d ownedin3d]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/ownedin3d">@ownedin3d</a> wrote:</p>
          <blockquote>
              <p>i have been working days now for finding out how this works, and finally i got it to work! and i was happy untill that a player used the command ::claim, wich made all the players of the server lagg out, and after 30 seconds they could log back in. does anyone know how to fix it?</p>
<p>[code]The last packet successfully received from the server was 1.344.662.672.572 mill<br>
iseconds ago.  The last packet sent successfully to the server was 0 millisecond<br>
s ago.<br>
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
0(Native Method)<br>
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
(NativeConstructorAccessorImpl.java:39)<br>
[11-08-12 07:24]:       at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(DelegatingConstructorAccessorImpl.java:27)<br>
[11-08-12 07:24]:       at java.lang.reflect.Constructor.newInstance(Constructor<br>
.java:513)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.SQLError.createCommunicationsException<br>
(SQLError.java:1074)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI<br>
mpl.java:2214)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.ConnectionImpl.(ConnectionImpl.j<br>
ava:773)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.JDBC4Connection.(JDBC4Connection<br>
.java:46)<br>
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
0(Native Method)<br>
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
(NativeConstructorAccessorImpl.java:39)<br>
[11-08-12 07:24]:       at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(DelegatingConstructorAccessorImpl.java:27)<br>
[11-08-12 07:24]:       at java.lang.reflect.Constructor.newInstance(Constructor<br>
.java:513)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionI<br>
mpl.java:352)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegist<br>
eringDriver.java:282)<br>
[11-08-12 07:24]:       at java.sql.DriverManager.getConnection(DriverManager.ja<br>
va:582)<br>
[11-08-12 07:24]:       at java.sql.DriverManager.getConnection(DriverManager.ja<br>
va:185)<br>
[11-08-12 07:24]:       at org.runetoplist.MySQLConnectionPool.getFreeConnection<br>
(MySQLConnectionPool.java:61)<br>
[11-08-12 07:24]:       at org.runetoplist.MySQLResultCallable.call(MySQLResultC<br>
allable.java:29)<br>
[11-08-12 07:24]:       at org.runetoplist.MySQLResultCallable.call(MySQLResultC<br>
allable.java:1)<br>
[11-08-12 07:24]:       at java.util.concurrent.FutureTask$Sync.innerRun(FutureT<br>
ask.java:303)<br>
[11-08-12 07:24]:       at java.util.concurrent.FutureTask.run(FutureTask.java:1<br>
38)<br>
[11-08-12 07:24]:       at java.util.concurrent.ThreadPoolExecutor$Worker.runTas<br>
k(ThreadPoolExecutor.java:886)<br>
[11-08-12 07:24]:       at java.util.concurrent.ThreadPoolExecutor$Worker.run(Th<br>
readPoolExecutor.java:908)<br>
[11-08-12 07:24]:       at java.lang.Thread.run(Thread.java:662)<br>
[11-08-12 07:24]: Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsExcep<br>
tion: Communications link failure</p>
<p>The last packet successfully received from the server was 1.344.662.672.572 mill<br>
iseconds ago.  The last packet sent successfully to the server was 0 millisecond<br>
s ago.<br>
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
0(Native Method)<br>
[11-08-12 07:24]:       at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
(NativeConstructorAccessorImpl.java:39)<br>
[11-08-12 07:24]:       at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(DelegatingConstructorAccessorImpl.java:27)<br>
[11-08-12 07:24]:       at java.lang.reflect.Constructor.newInstance(Constructor<br>
.java:513)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.SQLError.createCommunicationsException<br>
(SQLError.java:1074)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.MysqlIO.(MysqlIO.java:341)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI<br>
mpl.java:2137)<br>
[11-08-12 07:24]:       … 19 more<br>
[11-08-12 07:24]: Caused by: java.net.ConnectException: Connection timed out: co<br>
nnect<br>
[11-08-12 07:24]:       at java.net.PlainSocketImpl.socketConnect(Native Method)</p>
<p>[11-08-12 07:24]:       at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.ja<br>
va:351)<br>
[11-08-12 07:24]:       at java.net.PlainSocketImpl.connectToAddress(PlainSocket<br>
Impl.java:213)<br>
[11-08-12 07:24]:       at java.net.PlainSocketImpl.connect(PlainSocketImpl.java<br>
:200)<br>
[11-08-12 07:24]:       at java.net.SocksSocketImpl.connect(SocksSocketImpl.java<br>
:366)<br>
[11-08-12 07:24]:       at java.net.Socket.connect(Socket.java:529)<br>
[11-08-12 07:24]:       at java.net.Socket.connect(Socket.java:478)<br>
[11-08-12 07:24]:       at java.net.Socket.(Socket.java:375)<br>
[11-08-12 07:24]:       at java.net.Socket.(Socket.java:218)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.StandardSocketFactory.connect(Standard<br>
SocketFactory.java:253)<br>
[11-08-12 07:24]:       at com.mysql.jdbc.MysqlIO.(MysqlIO.java:290)<br>
[11-08-12 07:24]:       … 20 more<br>
[11-08-12 07:24]: java.util.concurrent.ExecutionException: java.lang.NullPointer<br>
Exception<br>
[11-08-12 07:24]:       at java.util.concurrent.FutureTask$Sync.innerGet(FutureT<br>
ask.java:222)<br>
[11-08-12 07:24]:       at java.util.concurrent.FutureTask.get(FutureTask.java:8<br>
3)<br>
[11-08-12 07:24]:       at org.runetoplist.VoteChecker.getReward(VoteChecker.jav<br>
a:40)<br>
[11-08-12 07:24]:       at server.model.players.packets.Commands.playerCommands(<br>
Commands.java:658)<br>
[11-08-12 07:24]:       at server.model.players.packets.Commands.processPacket(C<br>
ommands.java:52)<br>
[11-08-12 07:24]:       at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:137)<br>
[11-08-12 07:24]:       at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:3539)<br>
[11-08-12 07:24]:       at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:274)<br>
[11-08-12 07:24]:       at server.GameEngine.run(GameEngine.java:27)<br>
[11-08-12 07:24]:       at server.Server$1.execute(Server.java:183)<br>
[11-08-12 07:24]:       at server.task.Task.tick(Task.java:107)<br>
[11-08-12 07:24]:       at server.task.TaskScheduler.run(TaskScheduler.java:125)</p>
<p>[11-08-12 07:24]:       at java.util.concurrent.Executors$RunnableAdapter.call(E<br>
xecutors.java:441)<br>
[11-08-12 07:24]:       at java.util.concurrent.FutureTask$Sync.innerRunAndReset<br>
(FutureTask.java:317)<br>
[11-08-12 07:24]:       at java.util.concurrent.FutureTask.runAndReset(FutureTas<br>
k.java:150)<br>
[11-08-12 07:24]:       at java.util.concurrent.ScheduledThreadPoolExecutor$Sche<br>
duledFutureTask.access$101(ScheduledThreadPoolExecutor.java:98)<br>
[11-08-12 07:24]:       at java.util.concurrent.ScheduledThreadPoolExecutor$Sche<br>
duledFutureTask.runPeriodic(ScheduledThreadPoolExecutor.java:180)<br>
[11-08-12 07:24]:       at java.util.concurrent.ScheduledThreadPoolExecutor$Sche<br>
duledFutureTask.run(ScheduledThreadPoolExecutor.java:204)<br>
[11-08-12 07:24]:       at java.util.concurrent.ThreadPoolExecutor$Worker.runTas<br>
k(ThreadPoolExecutor.java:886)<br>
[11-08-12 07:24]:       at java.util.concurrent.ThreadPoolExecutor$Worker.run(Th<br>
readPoolExecutor.java:908)<br>
[11-08-12 07:24]:       at java.lang.Thread.run(Thread.java:662)<br>
[11-08-12 07:24]: Caused by: java.lang.NullPointerException<br>
[11-08-12 07:24]:       at org.runetoplist.PoolConnection.borrow(PoolConnection.<br>
java:27)<br>
[11-08-12 07:24]:       at org.runetoplist.MySQLConnectionPool.getFreeConnection<br>
(MySQLConnectionPool.java:66)<br>
[11-08-12 07:24]:       at org.runetoplist.MySQLResultCallable.call(MySQLResultC<br>
allable.java:29)<br>
[11-08-12 07:24]:       at org.runetoplist.MySQLResultCallable.call(MySQLResultC<br>
allable.java:1)<br>
[11-08-12 07:24]:       at java.util.concurrent.FutureTask$Sync.innerRun(FutureT<br>
ask.java:303)<br>
[11-08-12 07:24]:       at java.util.concurrent.FutureTask.run(FutureTask.java:1<br>
38)<br>
[11-08-12 07:24]:       … 3 more[/code]</p>
<p>i’ll pay 40M rsps gold, i go first but i must see that you have vouches.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-runetoplist-script-v2-pi/453807/1">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-runetoplist-script-v2-pi/453807/1</link>
        <pubDate>Sat, 11 Aug 2012 05:25:16 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-453807-1</guid>
        <source url="https://forum.moparisthebest.com/t/pi-runetoplist-script-v2-pi/453807.rss">[PI] Runetoplist script v2 [PI]</source>
      </item>
  </channel>
</rss>
